layouts/partials/hooks/body-end.html

```diff
@@ -2,3 +2,4 @@
   {{ partial "script.html" (dict "src" "js/tracing.js") -}}
 {{ end -}}
 {{ partial "script.html" (dict "src" "js/navScroll.js") -}}
+{{ partial "script.html" (dict "src" "js/stickySearch.js") -}}
```

Comment:
```suggestion
```

This will not be necessary anymore when you have moved the stickySearch.js to be included in the registrySearch.js

Reply:
Done!